Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN bundle install
 
 # Python dependencies for pywb
 WORKDIR /home/was/swap/current/pywb
-RUN pip3 install --requirement requirements.txt
+RUN pip3 install poetry
```

Comment:
Good question. I guess I could have left it the way it was, but when we upgrade to python 3.11 the `pip install` will go away entirely and be replaced with a `apt install python3-poetry` so we can bootstrap our virtual environment that way. I had been working on doing this until I realized pywb doesn't yet install under python v3.11 so I had to backup to 3.8, and didn't restore the `requirements.txt` in the process.

I don't think requests is needed with the `python:3.8` Docker image for reasons I don't totally understand. It's kind of weird that it was ever required since `pip install poetry` should have pulled it in...

Reply:
Sounds good!